cpp/ql/src/Likely Bugs/UnderspecifiedFunctions/TooFewArguments.c

```diff
@@ -0,0 +1,8 @@
+
+void one_argument(int x);
+
+void calls() {
+	one_argument(1); // GOOD: `one_argument` will accept and use the argument
+	
+	one_argument(); // BAD: `one_argument` will receive an undefined value
```

Comment:
This example doesn't compile because your declaration of `one_argument` has a prototype.